lib/v1/blogger/gratipay.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+This file is placed into Public Domain by
```

Comment:
As a suggestion, it might be better to use the CC0, since Public Domain isn't a thing in some countries.  I don't think it's a big enough issue to block though, so unless you want to change it @techtonik, I'll go ahead and merge it in a bit.